app/views/posts/show.html.erb

```diff
@@ -1,12 +1,16 @@
-<h1>Post Details</h1>
+<h1>Blog App</h1>
 
-<article class='no-posts'>
-  <div class='post-title'>
-    <h1><%= @post.title %> by <%= @post.author.name %></h1>
-    <div class='post-count-best'>Comments: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %></div>
+<main class='user-cont2'>
+  <div class="post-in-show">
+    <div class="post-in-show2">
+      <h3><%= @post.title %> by <%= @post.author.name %></h3>
+      <p>Comments: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %></p>
+    </div>
+    <p><%= @post.text %></p>
   </div>
-  <div><%= @post.text %></div>
-  <% @post.comments.each do |comment| %>
-    <%= render 'partials/comment', comment: comment %>
-  <% end %>
-</article>
+  <div class="comment-container">
+    <% @post.comments.each do |comment| %>
+      <%= render 'partials/comment', comment: comment %>
+    <% end %>
+  </div>
```

Comment:
- [ ] As requested by the previous code reviewer, Currently there is no like button on the post-show page to like a post so make sure that you have a like button on the post-show page as you have on the posts index page :blush: 
  
  ![image](https://github.com/BANCUNGUYE66/Blog-app/assets/92303891/aeece1a2-5dd5-43c3-b411-ab08ebd42977)